hazelcast/src/main/java/com/hazelcast/core/IMap.java

```diff
@@ -149,6 +155,30 @@
      * <p>
      * If you have previously set a TTL for the key, the TTL remains unchanged and the entry will
      * expire when the initial TTL has elapsed.
+     *
+     * <p><b>Interactions with the map store</b>
+     * <p>
+     * For each element not found in-memory
```

Comment:
"in-memory" -> "in memory". Dash is used if "in-memory" is used as an adjective. "an in-memory object", not "an object in memory".

Multiple instances of this are below.